src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.BuildTableDataSource.cs

```diff
@@ -193,6 +193,9 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         case StandardTableKeyNames.ErrorCodeToolTip:
                             content = GetHelpLinkToolTipText(data);
                             return content != null;
+                        case StandardTableKeyNames.HelpKeyword:
```

Comment:
@heejaechang Looking at the PR I realized I only added handler for SuppressionStateColumnDefinition.ColumnName in live source but not for build source, I will make that fix tomorrow.